Turbulence.ModelGenerator/Config.cs

```diff
@@ -0,0 +1,52 @@
+namespace Turbulence.ModelGenerator;
+
+// TODO: This should probably be a JSON/YAML file ¯\_(ツ)_/¯
+public static class Config
+{
+    /// <summary>
+    /// Root directory or URL for the .md files.
+    /// </summary>
+    public static readonly Uri DocsRoot = new("https://raw.githubusercontent.com/discord/discord-api-docs/main/docs");
+
+    /// <summary>
+    /// The path to temporarily store downloaded .md files at.
+    /// </summary>
+    public static readonly Uri TempPath = new($"{Directory.GetCurrentDirectory()}/../../../Out/Temp");
+    
+    /// <summary>
+    /// The path to store final models at.
+    /// </summary>
+    public static readonly Uri OutPath = new($"{Directory.GetCurrentDirectory()}/../../../Out/Models");
+    
+    /// <summary>
+    /// Location of .md files to generate models for, appended to the root directory.
+    /// </summary>
+    public static readonly List<string> MdFiles = new()
```

Comment:
Maybe instead just get all md files in the folder? If it isn't possible to get a proper file listing in a repo it should be okay enough for now

Reply:
I did it like this so you can choose to only regenerate from one file

Reply:
That feels like something no one wants to do though. The case that one file gets changed is there but who comes along and edits the config file for that